pkg/utils/storer_k8s.go

```diff
@@ -48,6 +49,7 @@ func connect2k8s(createTPR bool) (Proxy, ext_client.Clientset, error) {
 	}
 
 	clientset, err = proxy.SetupClientSet(config)
+	apiextensionsclientset, err := apiextensionsclient.NewForConfig(config)
```

Comment:
skipped the "err" from proxy.SetupClientSet

Reply:
and let's rename "createTPR" argument of "connect2k8s" func

Reply:
do we really need to use both clientsets (clientset and apiextensionsclientset) ?